### DISCUSSION TITLE
What is the release cadence for Large GitHub Runner Images?

### DISCUSSION THREAD

**Discussion by JBeresford** [actions/virtual-environments#8428] [General questions] [answered]
We have an issue in our pipeline which we believe is related to the **win22/20230903.2** image release. The step in question has been consistently failed for the last 3 weeks on our large runners which are seemingly pinned to this version.

It has been consistently working on on our builds which use the default runners on the following images:
* win22/20230910.1
* win22/20230918.1
* win22/20230924.1

When can we expect our large runner image to roll forward?

**Comment by Alexey-Ayupov** [comment, marked as answer]
Hello, @JBeresford. Indeed we had a problem with the Large runners image rollout.
We are going to make a new LR image release today with version `20230924.1`. It should be finished by the end of the day.

**Comment by matteovivona** [comment]
I'm jumping to this discussion. I also have a problem with the images preloaded by the large runner. I saw that a new version 20231001.1 was released already [3 days ago](https://github.com/actions/runner-images/releases/tag/ubuntu22/20231001.1) that contained a rollback of Chromium. However, when I run my workflows using a large runner with ubuntu-latest, the version is still the old one (20230924.1.0). 

![Screenshot 2023-10-05 at 14 45 46](https://github.com/actions/runner-images/assets/6388707/7ccfc314-220c-4d2e-bcd9-d06f80b7bb11)
![Screenshot 2023-10-05 at 14 45 25](https://github.com/actions/runner-images/assets/6388707/ae47fdb0-2c6c-487e-bfd0-fe569ad24eef)
